keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepositoryTests.kt

```diff
@@ -104,21 +104,21 @@ abstract class ArtifactRepositoryTests<T : ArtifactRepository> : JUnit5Minutests
     with(subject) {
       register(artifact1)
       setOf(version1, version2, version3).forEach {
-        store(artifact1, it, SNAPSHOT)
+        store(artifact1, it, SNAPSHOT, null)
```

Comment:
is it worth making the default value for this last parameter `null` so you don't have to update these calls?

Reply:
+1!

Reply:
done!